### DISCUSSION TITLE
Unable to Download app-release.apk build from Phone

### DISCUSSION THREAD

**Discussion by Aditya-QSS** [codemagic-ci-cd/codemagic-docs#2067] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I have triggered the build from Codemagic and I am able to download that build from my system but when I try to download it using my phone it shows Downloading... Download pending...

### Expected results

It should be downloadable from both the Phone and the System.

### Actual results

It is only getting downloaded from system not from phone.

**Comment by dtrdic** [comment]
hey @Aditya-QSS have you tried to run this apk on your local machine as well, does it run properly? please also share build id so I can take a closer look

**Reply by dtrdic** [reply]
Or do you mean that when you click on the download link using your phone, the binary doesn't download? Could you also share a screenshot to help me better understand the issue?

**Reply by Aditya-QSS** [reply]
Hello, @dtrdic It runs on my local machine and gets downloaded from my system. However, I am having issues while trying to download using my phone. Build Id:- 65116cde4056c01b61ed4242
![codemagic](https://github.com/codemagic-ci-cd/codemagic-docs/assets/92015923/24ca011a-f5d8-47b7-a1b5-f18dc93300ce)

**Reply by dtrdic** [reply]
can you please give me the exact steps, I can't reproduce the issue.. So when you click on the install button..

<img width="733" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/a9b20e95-7405-4b3d-a97c-ea6e1a44b464">

it redirects you to download the bundle, and when you click the download button it just stops on the downloading screen that you shared?

![IMG_2CA3666AF428-1](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/b1c865c3-3ab5-4996-82f9-4801e7e7e7aa)

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.

**Reply by gedeivanpratama** [reply]
I have a same issue "redirects me to download the bundle, and when you click the download button it just stops on the downloading screen

**Reply by dmytro-stepanov-hd** [reply]
@dtrdic I have the same issue. Try to download it on mobile, by scanning QA code for download app from phone.
It never starts the actual download.
desktop:
<img width="1055" alt="Screenshot 2024-10-24 at 10 08 18" src="https://github.com/user-attachments/assets/e53aed60-7a6d-47c1-b7a9-fdc81b59a15d">
mobile:
![image](https://github.com/user-attachments/assets/526a4c94-cae4-4eca-9fc8-62b64c5e3a43)
